benchmarks/covtype.py

```diff
@@ -0,0 +1,399 @@
+import argparse
+import os
+import subprocess
+import sys
+import time
+import warnings; warnings.simplefilter("ignore", FutureWarning)  # noqa: E702
+
+import numpy as onp
+
+# numpyro
+import jax
+from jax import numpy as np
+import numpyro
+import numpyro.distributions as ndist
+from numpyro.examples.datasets import COVTYPE, load_dataset
+
+# pyro
+import torch
```

Comment:
Just a note in case we face this issue in the future: with jaxlib 0.1.37, `import torch` might cause GPU memory error when running numpyro mcmc on this example (I don't have a reason for this and also can't come with an isolated code to raise upstream).